Document JOBDIR option (#5224)
scrapy/scrapy: docs/topics/settings.rst

A dict containing the pipelines enabled by default in Scrapy. You should never
modify this setting in your project, modify :setting:`ITEM_PIPELINES` instead.

.. setting:: JOBDIR

JOBDIR
------

Default: ``''``

> A string indicating the directory for storing the required data to keep the state of a single job to enable persistence support. This directory must not be shared by different spiders or jobs/runs of the same spider.
> 
> For more info on this setting, see :ref:`topics-jobs`

Reviewer: Thanks!

What would you think about shortening it a bit?

```suggestion
A string indicating the directory for storing the state of a crawl when
:ref:`pausing and resuming crawls <topics-jobs>`.
```

Author: I think it's much better, so it doesn't sound like we're repeating ourselves like a parrot.
